### PR TITLE
Use tc index for geckodriver downloads

### PR DESCRIPTION
**Pull request by jgraham**
Now we have a dedicated build+package task for geckodriver, we can
download the binaries from there

### CODE REVIEW

File: tools/wpt/browser.py [hunk 1]
```diff
@@ -49,6 +48,22 @@ def get_ext(filename):
...
+    TC_INDEX_BASE = FIREFOX_CI_ROOT_URL + "/api/index/v1/"
+
+    resp = get(TC_INDEX_BASE + "task/%s" % index)
+    resp.raise_for_status()
+    index = resp.json()
+    task_id = index["taskId"]
+
+    resp = get(TC_QUEUE_BASE + "task/%s/artifacts/%s" %
+               (task_id, path))
+    resp.raise_for_status()
```
Comment (catlee) [inline tools/wpt/browser.py:62]: There's no need to do both requests. You request the artifact directly from the index via a URL like this:
```
FIREFOX_CI_ROOT_URL + "/api/index/v1/task/%s/artifacts/%s" % (index, path)
```

Comment (jgraham) [reply tools/wpt/browser.py:62]: Woah TIL

**Review by catlee — APPROVED** [review]
LGTM!

**Comment by jgraham** [comment]
Failure is in Safari, so not related.

**Review by annevk — APPROVED** [review]

**Pull request merged**